cocos/webgpu/webgpu_wasm.js

```diff
@@ -0,0 +1,5151 @@
+
+var wasmDevice = (() => {
+  var _scriptDir = typeof document !== 'undefined' && document.currentScript ? document.currentScript.src : undefined;
```

Comment:
use `-` to connect words.

Reply:
this file is generated by emscripten, maybe ems's rules of format is different from us

Reply:
Ok